Content/Resources/en.lproj/Content.strings

```diff
@@ -0,0 +1,12 @@
+/* 
+  Content.strings
+  RealifeTech-SDK
+
+  Created by Mickey Lee on 02/03/2021.
+  Copyright © 2021 Realife Tech. All rights reserved.
+*/
+
+"ERROR" = "Error";
+"EMPTYURL_ERROR_MESSAGE" = "URL is empty";
+"CONSTRUCTURL_ERROR_MESSAGE" = "Unable to construct the URL";
```

Comment:
@SoaringEarth @rossrossp These error messages are used for `ContentError` in `ContentImplementing`. Do we need to localise copies in this phase? I've already set up the localisation for the Content interface, but it still can be removed if we don't need to do this. Could you confirm this please?

Reply:
Unless we have some similar already translated copy - I wouldn't worry about it for now it's good to keep the localisation though, as I would imagine eventually we will want to do this.